lerna.json

```diff
@@ -8,5 +8,6 @@
     "system-tests"
   ],
   "useWorkspaces": true,
+  "useNx": false,
```

Comment:
Once we get to a point where we can flip this switch we will start to see some significant DX and CI improvements! 😄